app/services/push_notifications/event_notifier/post_logic.rb

```diff
@@ -0,0 +1,24 @@
+module PushNotifications
+  class EventNotifier
+    class PostLogic
+      def initialize(event:)
+        @event = event
+        @post = event.post
+      end
+
+      # Conditions for Post:
+      #  - All the members of the associated organization
+      #  - that have a DeviceToken associated
```

Comment:
this second bullet point looks like the continuation of the previous one ⚠️ . It isn't needed, right?

Reply:
I'll rewrite this doc, it's an `AND` basically...